core-bundle/src/Resources/contao/dca/tl_page.php

```diff
@@ -191,7 +191,8 @@
 	(
 		'autoforward'                 => 'jumpTo',
 		'protected'                   => 'groups',
-		'includeLayout'               => 'layout',
+		'createSitemap'               => 'sitemapName',
```

Comment:
```suggestion
```

This looks like a left over from Contao 4.9?

Reply:
Good catch, thank you.